sonata-project/core-bundle/3.7/config/packages/sonata_core.yaml

```diff
@@ -0,0 +1,9 @@
+sonata_core:
+    flashmessage:
+        error:
+            css_class: danger
+            types:
+                - {type: sonata_product_error, domain: "SonataProductBundle" }
+        success:
+            types:
+                - {type: sonata_customer_success, domain: "SonataCustomerBundle" }
```

Comment:
To fix `The service "sonata.core.flashmessage.manager" has a dependency on a non-existent service "session".` error add this:

```yaml
framework:
    session: ~
```